packages/gatsby-cli/src/reporter/index.js

```diff
@@ -1,50 +1,47 @@
 // @flow
-
-const { createReporter } = require(`yurnalist`)
+const util = require(`util`)
 const { stripIndent } = require(`common-tags`)
-const convertHrtime = require(`convert-hrtime`)
+const chalk = require(`chalk`)
+const { trackError } = require(`gatsby-telemetry`)
 const tracer = require(`opentracing`).globalTracer()
 const { getErrorFormatter } = require(`./errors`)
-const { trackError } = require(`gatsby-telemetry`)
-
-const VERBOSE = process.env.gatsby_log_level === `verbose`
+const reporterInstance = require(`./reporters/yurnalist`)
```

Comment:
this is thight binding to yurnalist? you change this later to a setting?

Reply:
Yup, incrementally!